### PR TITLE
Box2D

### PR DESCRIPTION
**Pull request by MichaelHinrichs**
### SteamDB app page links to a few games using this
https://steamdb.info/app/342980/
https://steamdb.info/app/545820/
https://steamdb.info/app/855860/
https://steamdb.info/app/881100/
https://steamdb.info/app/1441850/

### Brief explanation of the change
Find Box2D physics library.

### CODE REVIEW

File: descriptions/SDK.Box2D.md [hunk 1]
```diff
@@ -0,0 +1 @@
+[**Box2D**](https://box2d.org/) is an [**open source**](https://github.com/erincatto/box2d) 2D Physics Engine for games. Written by Erin Catto.
```
Comment (Lifeismana) [inline descriptions/SDK.Box2D.md:1]:
```suggestion
[**Box2D**](https://box2d.org/) is an open source 2D Physics Engine for games. Written by Erin Catto.
```
No need to save pple a click, it's linked on their website

File: rules.ini [hunk 2]
```diff
@@ -185,6 +185,8 @@ Anzu = (?:^|/)anzu\.dll$
 Azure_Playfab_Party = (?:^|/)Party(?:Win|Win7|Win32|XboxLive|XboxLiveWin32)\.dll$
 BASS = (?:^|/)(?:bass(?:flac|midi)?\.dll|libbass\.(?:dylib|so))$
 Bink_Video = (?:^|/)bink2?w(?:64|32)?\.dll$
+Box2D[] = (?:^|/)box2d\.(?:(?:XNA\.)?dll|txt|wasm(?:\.js)?)$
+Box2D[] = (?:^|/)libs/gdx-box2d
```
Comment (Lifeismana) [inline rules.ini:189]:
```suggestion
Box2D[] = (?:^|/)box2d\.(?:(?:XNA\.)?dll|txt|wasm)$
```
second one is for one game only so won't add

**Review by Lifeismana — CHANGES_REQUESTED** [review]

**Comment by Lifeismana** [comment]
missing non matching tests

**Comment by MichaelHinrichs** [comment]
Done.

**Comment by SteamTracker** [comment]
After running changed rules in this pull request *(this bot only tests rules.ini changes)*:

Changed technologies: **SDK.Box2D**

These apps now match: **(78)** [11360](https://steamdb.info/app/11360/depots/) [212700](https://steamdb.info/app/212700/depots/) [213390](https://steamdb.info/app/213390/depots/) [224660](https://steamdb.info/app/224660/depots/) [251510](https://steamdb.info/app/251510/depots/) [311940](https://steamdb.info/app/311940/depots/) [322190](https://steamdb.info/app/322190/depots/) [337110](https://steamdb.info/app/337110/depots/) [342980](https://steamdb.info/app/342980/depots/) [391150](https://steamdb.info/app/391150/depots/) [400200](https://steamdb.info/app/400200/depots/) [406130](https://steamdb.info/app/406130/depots/) [444750](https://steamdb.info/app/444750/depots/) [454410](https://steamdb.info/app/454410/depots/) [545820](https://steamdb.info/app/545820/depots/) [571310](https://steamdb.info/app/571310/depots/) [585410](https://steamdb.info/app/585410/depots/) [585600](https://steamdb.info/app/585600/depots/) [585610](https://steamdb.info/app/585610/depots/) [585620](https://steamdb.info/app/585620/depots/) [645200](https://steamdb.info/app/645200/depots/) [685420](https://steamdb.info/app/685420/depots/) [713310](https://steamdb.info/app/713310/depots/) [804010](https://steamdb.info/app/804010/depots/) [855860](https://steamdb.info/app/855860/depots/) [866710](https://steamdb.info/app/866710/depots/) [881100](https://steamdb.info/app/881100/depots/) [951170](https://steamdb.info/app/951170/depots/) [954550](https://steamdb.info/app/954550/depots/) [1008860](https://steamdb.info/app/1008860/depots/) [1066900](https://steamdb.info/app/1066900/depots/) [1103790](https://steamdb.info/app/1103790/depots/) [1210640](https://steamdb.info/app/1210640/depots/) [1334630](https://steamdb.info/app/1334630/depots/) [1401500](https://steamdb.info/app/1401500/depots/) [1428840](https://steamdb.info/app/1428840/depots/) [1438260](https://steamdb.info/app/1438260/depots/) [1441850](https://steamdb.info/app/1441850/depots/) [1443400](https://steamdb.info/app/1443400/depots/) [1448210](https://steamdb.info/app/1448210/depots/) [1453030](https://steamdb.info/app/1453030/depots/) [1490560](https://steamdb.info/app/1490560/depots/) [1491760](https://steamdb.info/app/1491760/depots/) [1499210](https://steamdb.info/app/1499210/depots/) [1610660](https://steamdb.info/app/1610660/depots/) [1627820](https://steamdb.info/app/1627820/depots/) [1662860](https://steamdb.info/app/1662860/depots/) [1670460](https://steamdb.info/app/1670460/depots/) [1688370](https://steamdb.info/app/1688370/depots/) [1702460](https://steamdb.info/app/1702460/depots/) [1710460](https://steamdb.info/app/1710460/depots/) [1778430](https://steamdb.info/app/1778430/depots/) [1794190](https://steamdb.info/app/1794190/depots/) [1819430](https://steamdb.info/app/1819430/depots/) [1819890](https://steamdb.info/app/1819890/depots/) [1861680](https://steamdb.info/app/1861680/depots/) [1884310](https://steamdb.info/app/1884310/depots/) [1999680](https://steamdb.info/app/1999680/depots/) [2001670](https://steamdb.info/app/2001670/depots/) [2019150](https://steamdb.info/app/2019150/depots/) [2089170](https://steamdb.info/app/2089170/depots/) [2143930](https://steamdb.info/app/2143930/depots/) [2183010](https://steamdb.info/app/2183010/depots/) [2299270](https://steamdb.info/app/2299270/depots/) [2302100](https://steamdb.info/app/2302100/depots/) [2309630](https://steamdb.info/app/2309630/depots/) [2398960](https://steamdb.info/app/2398960/depots/) [2477090](https://steamdb.info/app/2477090/depots/) [2559290](https://steamdb.info/app/2559290/depots/) [2575150](https://steamdb.info/app/2575150/depots/) [2598410](https://steamdb.info/app/2598410/depots/) [2613010](https://steamdb.info/app/2613010/depots/) [2628870](https://steamdb.info/app/2628870/depots/) [2648520](https://steamdb.info/app/2648520/depots/) [2656430](https://steamdb.info/app/2656430/depots/) [2663470](https://steamdb.info/app/2663470/depots/) [2694140](https://steamdb.info/app/2694140/depots/) [2752950](https://steamdb.info/app/2752950/depots/)

Please confirm that all these changed apps are correct.

**Comment by Lifeismana** [comment]
merged in a05d79a79e58a39981ef38bb4591bbc52035899b

**Pull request closed without merging**